contracts/eip20/EIP20Interface.sol

```diff
@@ -43,6 +44,7 @@ contract EIP20Interface {
     /// @return Amount of remaining tokens allowed to spent
     function allowance(address _owner, address _spender) public view returns (uint256 remaining);
 
-    event Transfer(address indexed _from, address indexed _to, uint256 _value);
+    // solhint-disable-next-line no-simple-event-func-name  
```

Comment:
Just curious, what is solhint mad about here?

Reply:
`Event and function names must be different` https://protofire.github.io/solhint/rules.html

Seems like a good rule, but EIP20 compliance breaks it.